.github/workflows/cd.yml

```diff
@@ -133,6 +133,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```

Comment:
because we need to know revision for merge commit of pull-request. Actually it gonna be HEAD^.